### DISCUSSION TITLE
Como participar de comunidades de tecnologia te ajudou?

### DISCUSSION THREAD

**Discussion by felipebernardes** [frontendbr/forum#290] [Discussão]
### Oi, Pessoal

Comecei uma pesquisa pra descobrir o impacto de comunidades de tecnologia nas pessoas. 

Além disso, quero escrever um post pra incentivar quem hoje não participa de comunidades de tecnologia, passarem a participar.

Peço que contribuam respondendo (são só quatro perguntinhas) e divulgando prozamigo.

[link da pesquisa (google forms)](https://docs.google.com/forms/u/0/d/1pegJ1xShtOKR9JJu6q5XBcXEAl6eQ5JfjI4uUqu4MZk)

Valeu!! #frontendbr :facepunch:

EDIT: 
O @Mendrone deu uma idéia muito legal:

> Se alguém, além de responder a pesquisa, quiser também deixar um depoimento por aqui seria bem legal :grin:

**Comment by vitor-mendrone-deel** [comment]
Respondido!

**Comment by grippado** [comment]
Respondido!

**Comment by vitor-mendrone-deel** [comment]
Se alguém, além de responder a pesquisa, quiser também deixar um depoimento por aqui seria bem legal 😁

**Comment by felipebernardes** [comment]
Obrigado, pessoal! @Mendrone , excelente idéia. Vou editar o post e incluí-la.

**Comment by Kirmayrtomaz** [comment]
Done
